src/TSMapEditor/UI/Windows/TeamTypesWindow.cs

```diff
@@ -325,9 +330,21 @@ private void EditTeamType(TeamType teamType)
             tbPriority.Value = editedTeamType.Priority;
             tbMax.Value = editedTeamType.Max;
             tbTechLevel.Value = editedTeamType.TechLevel;
+
+            if (Constants.UseCountries)
+            {
+                ddMindControlDecision.SelectedIndex = editedTeamType.MindControlDecision ?? -1;
+            }
+
             tbGroup.Value = editedTeamType.Group;
+
             tbWaypoint.Value = Helpers.GetWaypointNumberFromAlphabeticalString(editedTeamType.Waypoint);
 
+            if (Constants.UseCountries)
+            {
+                tbTransportWaypoint.Value = Helpers.GetWaypointNumberFromAlphabeticalString(editedTeamType.TransportWaypoint);
+            }
```

Comment:
Looks better like this IMO, also no need to repeat the condition.

```suggestion
            tbGroup.Value = editedTeamType.Group;
            tbWaypoint.Value = Helpers.GetWaypointNumberFromAlphabeticalString(editedTeamType.Waypoint);

            if (Constants.UseCountries)
            {
                ddMindControlDecision.SelectedIndex = editedTeamType.MindControlDecision ?? -1;
                tbTransportWaypoint.Value = Helpers.GetWaypointNumberFromAlphabeticalString(editedTeamType.TransportWaypoint);
            }
```